### DISCUSSION TITLE
Group Policy Settings

### DISCUSSION THREAD

**Discussion by Snuggles266** [InternetConnectionMonitor/icm-public#102] [Q&A] [answered]
I deployed ICM to my company's fleet of laptops and have been successful so far in the deployment.  I am reaching out to ask if there is any documentation on how to set the settings using Group Policies.  I want to be able to PIN the extension in their browser and also make changes to the settings as my company uses it.  Thanks for any help.

**Comment by dmishh** [comment]
@Snuggles266 I'm not aware of the enterprise capabilities for Google Chrome. The extension itself doesn't have an option to lock with a PIN

**Comment by dmishh** [comment, marked as answer]
@Snuggles266 Sorry, I misread your question. You probably should be looking for some docs about setting the group policies externally from the extension itself. Like here https://support.google.com/chrome/a/answer/7532015?hl=en
